share/ert/forward-models/res/eclipse100

```diff
@@ -1 +1,5 @@
 EXECUTABLE script/ecl100
+DEFAULT <VERSION> version
```

Comment:
In my opinion these jobs should not use any of the keywords besides the `EXECUTABLE`.

Reply:
you are absolutely right, i screwed up while reverting